formats/email_alert_signup/publisher/details.json

```diff
@@ -13,17 +13,7 @@
       "minProperties": 1,
       "properties": {
         "tags": {
-          "type": "object",
-          "description": "DEPRECATED: The tags used to match subscribers lists",
```

Comment:
I think that description is good, I guess I was concerned by the "DEPRECATED" bit from here which we've lost (I assume this has some semantic meaning about these formats where publisher will stop sending tags soon?) and the description of tags on hmrc_manuals which doesn't indicate its got anything to do with email-alert-api (although it may do - I'm just not sure).